examples/src/main/scala/za/co/absa/enceladus/examples/CustomRuleSample3.scala

```diff
@@ -29,11 +29,9 @@ object CustomRuleSample3 {
     .appName("CustomRuleSample3")
     .config("spark.sql.codegen.wholeStage", value = false)
     .getOrCreate()
-
-  spark.sparkContext.setLogLevel("WARN")
+  TimeZoneNormalizer.normalizeAll(Seq(spark)) //normalize the timezone of JVM and ALL the spark sessions (one in this case)
```

Comment:
I think the `Seq` is obsolete now.
```suggestion
  TimeZoneNormalizer.normalizeAll(spark) //normalize the timezone of JVM and ALL the spark sessions (one in this case)
```

Reply:
It's a sample code. Shows options. I hoped the comment was explaining it.

Reply:
I am not sure about that, it seems like it shows something that should not be used. But I don't have a strong preference for keeping it like this or not, so I am leaving it to you.